amplipi/ctrl.py

```diff
@@ -352,7 +352,7 @@ def _is_digital(self, sinput: str) -> bool:
     """
     try:
       sid = int(sinput.replace('stream=',''))
-      return sid in RCAs
+      return sid not in RCAs
     except:
       return False
```

Comment:
Can this return True as a default instead? I'm not sure what would cause an exception there but Digital is technically the safer default from a hardware perspective. Since the Analog RCA inputs are floating, random noise will be on any unconnected RCA inputs.

Reply:
Yup, that's what it should do!